repo_policy_compliance/check.py

```diff
@@ -38,11 +38,6 @@
     f"include the string '{AUTHORIZATION_STRING_PREFIX} <commit SHA>' where the commit SHA is the "
     "SHA of the latest commit on the branch"
 )
-ERROR_MESSAGE = (
-    "\n"
-    "Something went wrong  while checking repository compliance policy. "
-    "Please contact the operator."
-)
```

Comment:
I think this error message is better than `The github client encountered an error` for an end user. The end user may have no knowledge of our mechanism and that we use a github client.

Reply:
Updated!